### ISSUE TITLE
Crash at start 

### ISSUE THREAD

**Issue by netshaman** [KozGit/DOOM-3-BFG-VR#160]
The game crashes when starting in VR, without any message or such , is there somewhere an error log or whatever ?

**Comment by KozGit** [comment]
Create a shortcut to Doom3BFGVR.exe
Right click on the shortcut, go to properties.
In the target field, add +set logFile 2 to the end - be sure to leave a space after .exe and before +set
Click apply, then ok.
Double click shortcut to launch the game.

The log file will be stored in c:\users\(your user name)\saved games\id software\doom 3 bfg\base\qconsole.log

**Comment by netshaman** [comment]
Thank you

**Comment by netshaman** [comment]
Ok done, and thre is some warnings in the log file that says this kind of thing :" line 4250: material 'vr/statwatch' previously defined at materials/vr.mtr:92
WARNING: file materials/weapons.mtr, line 4258: material 'vr/statwatchgui' previously defined at materials/vr.mtr " As i can  see there is some discrepancies with the names of the materials wich were previously defined, but where are they ?

**Comment by KozGit** [comment]
Yeah, those warnings are normal, just some of the material files need to be cleaned up - they shouldn't stop the game from launching.  Can you attach the whole log file here?

**Comment by narfff203** [comment]
My game also crashes when starting up in VR.
I use a OSVR HDK2 could that be the reason?

**Comment by KozGit** [comment]
I'm not very familiar with the OSVR HDK2.  Does SteamVR support it?  If not, the mod will not work unfortunately.  If SteamVR supports the headset, attach the output of the logfile ( see instructions above ).

**Comment by netshaman** [comment]
Hi , sorry for the long delay , here is the config file:

https://www.dropbox.com/s/ocrtlgdlw6wuae8/qconsole.log?dl=0

**Comment by narfff203** [comment]
Hi there,

Sorry for the late respons, I added the log in attachments.
I hope you can help :)

Thank you



2017-06-14 14:35 GMT+01:00 netshaman <notifications@github.com>:

**Comment by netshaman** [comment]
I had deinstalled/reinstalled the game, made a fresh install of the mod without any  texture pack to see what happens, and still crashing :(

**Comment by KozGit** [comment]
Hi, its strange, the log file just ends at 'skipping intro videos', no failures or errors listed.  What happens on you monitor when it crashes? any messages at all? does it drop to the desktop or to the game console?

Also, what type/how many graphics cards do you have?

**Comment by MrImmaculate** [comment]
I'm seeing the same crash. My log ends at the same point... "skipping intro videos"

All it gives me is a dialog Doom3 BFG has crashed

Win10 Pro 64-bit v10.0.14393
i7-3770
16gb
500gb 850 evo SSD
GTX 1080 v382.53
Oculus Home v1.16

I'm in the process of reinstalling the game, mod, and updating my gpu drivers.

**Comment by MrImmaculate** [comment]
Nvidia 385.12 doesn't help...

https://www.dropbox.com/s/dtmi8vsu4z1mebm/qconsole.log?dl=0

![d3fp error](https://user-images.githubusercontent.com/30679632/28905472-d68a07ee-77de-11e7-8fb9-42dee17898fb.png)
